ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -1397,6 +1430,7 @@ private static void checkBlobId(Router router, BlobId blobId, byte[] data) throw
 
   private static void checkBlobContent(MockClusterMap clusterMap, BlobId blobId, BlockingChannel channel,
       byte[] dataToCheck) throws IOException, MessageFormatException {
+    // todo: fix once router support encryption. Might have to decryt to compare data
```

Comment:
"decrypt"

Reply:
same here - how would this happen since there is no router involvement in these tests?

Reply:
yes, you are right. No decryption is required. Will remove the TODOs for now. Will revisit in my other patch, how we can add encryption tests here.